deploy/servctl_utils/deploy_command_utils.py

```diff
@@ -211,9 +211,6 @@ def deploy_external_connector(settings, connector_name):
 def deploy_elasticsearch(settings):
     print_separator("elasticsearch")
 
-    if settings["DELETE_BEFORE_DEPLOY"]:
-        delete_pod("elasticsearch", settings)
-
```

Comment:
Currently all the deploy helper functions call this conditional delete, than deploy docker, then deploy the pod. We want to wait and delete the pod until after the docker deploy is done to minimize downtime, but rather than moving this if block around 6 time I just moved it into `deploy_pod` so theres less copy-pasting

moved this into `deploy_pod` since every time that function is called this needs to be too and this way theres less copy-paste. Plu